get_id.js

```diff
@@ -1,104 +1,164 @@
-
-let Url = "http://164.68.125.44:34351/table/items/get?limit=1&lang=1&id=2"
-let local = JSON.parse(localStorage.USER);
-let token1 = local["token"]
-
-let response = await fetch(Url, {
-    headers: {
-        Authorization: "Bearer " + token1
+async function getEP_write_placeholders (id) {
+    let resp
+    let port = window.location.origin
+    let Url = `${port}/table/items/get?limit=1&lang=1&id=${id}`
+    let local = JSON.parse(localStorage.USER);
+    let token1 = local["token"]
+    let response = await fetch(Url, {
+        headers: {
+            Authorization: "Bearer " + token1
+        }
+    });
+    if (response.status === 200) {
+        resp = await response.json()
     }
-  });
-
-if (response.ok) {
-    resp = await response.json()
-}
-resp = resp[0]
-
-// let resp = response.json()
-console.log(resp)
-let id_type_material = resp.id_type_material
-let id_type_of_filler
-console.log(id_type_material)
-
-let filler_dispers = ["marka", "manufacturer", "chemical_nature", "chemical_nature_iso", "purity_of_disperse_filler", "purity_of_disperse_filler_iso", "dispersity", "dispersity_iso", "moisture_content", "moisture_content_iso", "density", "density_iso"]
-
-let filler_fibrous = ["marka", "manufacturer", "density", "density_iso", "linear_density_iso", "linear_density", "length_of_fibres_iso", "length_of_fibres", "filament_diameter", "filament_diameter_iso", "tensile_strength_iso", "tensile_strength", "tensile_modulus", "tensile_modulus", "elongation_break", "elongation_break_iso", "moisture_content", "moisture_content_iso"]
-
-let filler_other = ["marka", "manufacturer", "chemical_nature", "chemical_nature_iso", "purity_of_disperse_filler", "purity_of_disperse_filler_iso", "dispersity", "dispersity_iso", "moisture_content", "moisture_content_iso", "density", "density_iso", "content_filler_iso", "content_filler"]
-
-let TKM_disperse_filler = ["marka", "chemical_nature", "content_reinforcement_iso", "content_reinforcement", "density_iso", "density", "mfr_iso", "mfr", "tensile_modulus_iso", "tensile_modulus", "tensile_strength_break_iso", "tensile_strength_break", "linear_density_iso", "linear_density", "length_of_fibres_iso", "length_of_fibres", "filament_diameter_iso", "filament_diameter", "tensile_strength_iso", "tensile_strength", "elongation_break_iso", "elongation_break", "charpy_notched_impact_strength", "charpy_impact_strength", "melting_temperature", "moisture_content_iso", "moisture_content"]
-
-if (id_type_material === 1) {
-    console.log("Thermoplastic polymer")
-} else if (id_type_material === 2) {
-    console.log("Additives")
-} else if (id_type_material === 3) {
-    console.log("Thermoplastic elastomer")
-    if (id_type_of_filler === 1) {
-        console.log("Thermoplastic composite materials => Disperse filler")
-        parse_TKM_disperse_filler(resp, TKM_disperse_filler)
-    } else if (id_type_of_filler === 2) {
-        console.log("Thermoplastic composite materials => Fibrous filler")
+    resp = resp[0]

```

Comment:
We will get error here when response.status != 200. 
You must add handling of other status of response. 
Example, 204 - not found, 400 - bad params. 
Func continue ONLY when status = 2000